### ISSUE TITLE
Not GPL compliant

### ISSUE THREAD

**Issue by l0k1** [FGMEMBERS/SpaceShuttle#3]
Hey,
This repo is not licensed under GPL, nor is it currently GPL compliant. I'd recommend deletion of the repo.

**Comment by FGDATA** [comment]
The repository IS GPL. The Space Shuttle you see here is derived from GPL work initially by Jon S. Berndt (the creator of JSBsim) and then modified on GPL clauses by Herbert Wagner (HerbyW), and finally continued work by Thorsten, Wllbragg, Zaretto and others.
[Copy left clauses](http://www.gnu.org/licenses/copyleft.en.html) left beforehand by J.S.B, and HerbyW, prevent Thorsten from legally claim this is his copyrighted content lacking a license.

True is that Copyleft clauses force Thorsten to maintain all derivative work bundled here as GPL, or he will be violating both J.S.Bernd, and HerbyW copyrights. 

The fact that a [public repository ](http://sourceforge.net/p/fgspaceshuttledev/code/ci/master/tree/)of Thorsten Work exists in Sourceforge (where many others clone and use) is directly considered as a distribution, and Thus, forces Thorsten to comply with the GPL license that applied before he began working with this airplane.

Best,
IAHM-COL

**Comment by FGDATA** [comment]
https://github.com/FGMEMBERS/shuttle

Initial commit by Curt on older FGDATA https://github.com/FGMEMBERS/shuttle/commit/422e1838e97bd7542540d32de20a4e73fd13c717
(July 13,2001)

First commit by Thorsten Renk in sourceForge derivating work 
https://github.com/FGMEMBERS/SpaceShuttle/commit/9ce0b1f3183a45c225800dae3f573c71f5a507c1
(April 5 , 2015)

In the initial commit we can read the original Copyright owner who granted GPL to this work and any derivative material

https://github.com/FGMEMBERS/shuttle/blob/422e1838e97bd7542540d32de20a4e73fd13c717/shuttle.xml

Line 5 : Jon S. Berndt

**Comment by FGDATA** [comment]
The matter of the issue is summarized here
http://www.gnu.org/licenses/copyleft.en.html

>  Copyleft also helps programmers who want to contribute improvements to free software get permission to do so. These programmers often work for companies or universities that would do almost anything to get more money. A programmer may want to contribute her changes to the community, but her employer may want to turn the changes into a proprietary software product.
> 
> **When we explain to the employer that it is illegal to distribute the improved version except as free software**, the employer usually decides to release it as free software rather than throw it away.
> 
> To copyleft a program, we first state that it is copyrighted; then we add distribution terms, which are a legal instrument that gives everyone the rights to use, modify, and redistribute the program's code, or any program derived from it, **but only if the distribution terms are unchanged**. Thus, the code and the freedoms become legally inseparable.

**Comment by l0k1** [comment]
Firstly,
Holy Buddha calm your tits.
Secondly, after reviewing the space shuttle thread, Thorsten released this as GPL, so meh.

**Comment by FGDATA** [comment]
ok. Thanks for revising thou.
You'll see this is an old tire-thread and it is really slippery stuff. Thorsten has been using this fabrication as leverage to create an ill-informed view of me and FGMEMBERS, purely as propaganda, and you may see him coming back arguing that I can't call Copy-Left on him, for some reason that scape Buddha or any other deity or or profet grasp. Off course, except his own grasp. 
You may have lost his new 'explaining GPL in 60 steps he manufactured' in the Devel - List, which is mostly a correct document, but intended at mis-spreading information. 
In other words, yeah, I apologize explaining this on thorough detail, but I am certain we had not seen the end of Thorsten on this one: Regardless of the weird the whole thing results; as in "It is not GPL for FGMEMBERS, but it is for anyone else"

Have a great day l0k1.
